frontend/src/views/Review.vue

```diff
@@ -53,6 +53,12 @@ export default {
   },
   methods: {
     submitReply (replyForm) {
+      const authState = this.$store.getters['auth/isLoggedIn']
```

Comment:
Don't do this. We're going to remove the functionality. And prompt, not force them.

Reply:
Same with above.

Reply:
Do we have hidden prompt functionality or should I implement this now?

Reply:
There's not presently. I was thinking button would be replaced with text if user is not authenticated. "You need to be logged in to write a review" etc OR button goes to disabled state and has a hover text.

Reply:
That's a lot of text for a button. The second option might be viable. I'll have a play and see what feels natural :)

Reply:
It wouldn't be a button.

Reply:
^